buildconfig/CMake/MSVCSetup.cmake

```diff
@@ -176,7 +176,33 @@ file(
 set(MANTIDPYTHON_PREAMBLE
     "set PYTHONHOME=%_BIN_DIR%\nset PATH=%_BIN_DIR%;%_BIN_DIR%\\..\\plugins;%_BIN_DIR%\\..\\PVPlugins;%PATH%"
 )
+# Launch script
+# Developer verison
+set(MANTIDLAUNCH_PREAMBLE
```

Comment:
Nice. This should mean we don't have to use the `command-prompt` to start workbench.